src/Common/src/Internal/Runtime/TypeLoader/EETypeFlags.cs

```diff
@@ -101,4 +101,58 @@ internal enum EETypeKind : ushort
         /// </summary>
         GenericTypeDefEEType = 0x0003,
```

Comment:
I'll change it back - EEType related code we share from the TypeLoader will need adjusting to use Internal.Runtime as it is brought over but that should be fine.